test/testutil/driver.c

```diff
@@ -150,10 +146,29 @@ void set_test_title(const char *title)
     test_title = title == NULL ? NULL : strdup(title);
 }
 
+static void test_verdict(int pass, const char *extra, ...) PRINTF_FORMAT(2, 3);
+static void test_verdict(int pass, const char *extra, ...)
```

Comment:
Since it's static, there's no need for the declaration before the definition, just merge these two lines into one.